tracing-fmt/src/default.rs

```diff
@@ -181,7 +191,7 @@ impl<'a> field::Visit for Recorder<'a> {
     }
 }
 
-struct FmtCtx<'a, N: 'a>(&'a span::Context<'a, N>);
+struct FmtCtx<'a, N: 'a>(&'a span::Context<'a, N>, bool);
```

Comment:
Nit: if this is going to have two fields, I'd really prefer it if it was a regular struct with named fields rather than a tuple struct.